packages/web-components/src/components/countdown/__stories__/countdown.stories.ts

```diff
@@ -0,0 +1,97 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html } from 'lit-element';
+import '../index';
+import ifNonEmpty from '@carbon/web-components/es/globals/directives/if-non-empty';
+import { bxGrid16Col } from '../../../globals/internal/storybook-decorators';
+
+import readme from './README.stories.mdx';
+
+const msInDay = 86400000;
+const twoWeeksFromNowTimestamp =
+  Number(new Date().getTime().toString()) + 14 * msInDay;
+const twoWeeksFromNowISO = new Date(twoWeeksFromNowTimestamp).toISOString();
+
+export default {
+  title: 'Components/Countdown',
+  parameters: {
+    ...readme.parameters,
+  },
+  argTypes: {
+    targetDate: {
+      control: { type: 'date' },
+      name: 'Target Date',
+      defaultValue: twoWeeksFromNowISO,
+    },
+    separator: {
+      control: { type: 'text' },
+      name: 'Separator',
+      defaultValue: ', ',
+    },
+    labelType: {
+      control: { type: 'select' },
+      options: ['long', 'short', 'narrow', 'none'],
+      name: 'Label Type',
+      defaultValue: 'long',
+    },
+  },
```

Comment:
These controls are not showing up in Storybook. I believe you need to convert these to knobs. See other functioning stories with with knobs in the repo.